app/tps_counter.go

```diff
@@ -0,0 +1,72 @@
+package app
+
+import (
+	"context"
+	"sync/atomic"
+	"time"
+
+	"github.com/tendermint/tendermint/libs/log"
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+)
+
+var mTransactions = stats.Int64("transactions", "the number of transactions after .EndBlocker", "1")
+var viewTransactions = &view.View{
+	Name:        "evmos/app",
+	Measure:     mTransactions,
+	Description: "The transactions processed",
+	TagKeys:     nil,
+	Aggregation: view.Count(),
+}
+
+type tpsCounter struct {
+	nTxn         uint64
```

Comment:
I purposefully didn't use big.Int because, to use maxUint64 in a short time, that'd require like processing quintillion requests highest of 18446744073709551615 per second in which case is physically impossible, but also this code is meant to be super fast without adding overhead in the very fast path. Using big.Int would be so expensive unnecessarily. How does that sound?